.github/labeler.yml

```diff
@@ -0,0 +1,39 @@
+algorithms:
+  - src/pymortests/algorithms/*
+  - src/pymor/algorithms/*
+
+buildsystem:
+  - .ci/*
+  - .github/*
+  - .binder/*
+  - Makefile
+  - azure-pipelines.yml
+  - requirements*
+  - pyproject.toml
+  - MANIFEST.in
+  - dependencies.py
+  - setup.*
+  - versioneer.py
+
+bindings:
+  - src/pymor/bindings/*
+
+builtin-models:
+  - src/pymor/discretizers/*
+  - src/pymor/playground/discretizers/*
+  - src/pymortests/model.py
+  - src/pymortests/fixtures/model.py
+  - src/pymor/models/*
+
+docs:
+  - docs/*
```

Comment:
This would add the 'docs' tag even if it is just a code update in the tutorial to account for e.g. for example.

Reply:
Not sure I understand what you want to say. If the PR _only_ updates code in a tutorial's code block then this is not a documentation update?

Reply:
Yes, if I make an API change which breaks the code in the tuorials, and the only thing the PR does to the documentation is fixing that, I do not see what why one would want to add the 'docs' label.